content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-13.md

```diff
@@ -33,18 +33,38 @@ This page explains how to upgrade a Kubernetes cluster created with `kubeadm` fr
 
 {{% capture steps %}}
 
-## Upgrade the control plane
+## Determine what version you want to upgrade to
```

Comment:
nit, optional but good to fix while we wait for lgtm:
-> ## Determine which version to upgrade to